### DISCUSSION TITLE
Backpack AddOn : Subscriptions Managment

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#1138] [Ideas]
I'm looking for an addon to manage subscription plans for my apps, any ideas?

Thanks for help

**Comment by karandatwani92** [comment]
Hey @blondie63 

I've not seen any add-on for the subscription. If you are going to code it for your requirements, you can choose to make it as a Backpack Add-on. This way, you use it and share your work with the community.

Here are some resources to create a Backpack Add-on:
- https://backpackforlaravel.com/docs/6.x/add-ons-tutorial-using-the-addon-skeleton
- https://backpackforlaravel.com/articles/tutorials/how-to-create-a-backpack-add-on-with-a-custom-operation

If not, you can suggest the idea [here](https://github.com/Laravel-Backpack/community-forum/discussions/751). If the masses become interested, we may develop it later.
